crates/driver/src/domain/mempools.rs

```diff
@@ -123,24 +128,32 @@ impl Mempools {
                     TxStatus::Pending => {
                         // Check if the current block reached the submission deadline block number
                         if block.number >= submission_deadline {
+                            let cancellation_hash = self
+                                .cancel(mempool, settlement.gas.price, solver)
+                                .await
+                                .context("cancellation tx failed")?;
                             tracing::info!(
                                 ?hash,
```

Comment:
given that we now have 2 hashes in the same log I think we should give this one a better name.